docs/pages/link.js

```diff
@@ -8,44 +8,281 @@ import GeneratedPropTable from '../components/GeneratedPropTable.js';
+    </Box>
+    <TextField
+      placeholder="Create password"
+      id="example-password-1"
+      onChange={() => {}}
+      type="password"
+      value=""
+    />
+  </Flex.Item>
+  <Flex.Item>
+    <Box display="visuallyHidden">
+      <Label htmlFor="example-age-1">Age</Label>
+    </Box>
+    <NumberField placeholder="Age" id="example-age-1" onChange={() => {}} value="" />
+  </Flex.Item>
+  <Button fullWidth text="Create account" size="md" color="red" />
+  <Button fullWidth text="Log into existing account" size="md" type="submit" />
+  <Text size="sm" align="center">
+    By continuing, you agree to Pinterest's
+    <Text size="sm" inline>
+      <Link href="https://www.pinterest.com" inline>
+        {' '}
+        Business Terms of Service{' '}
+      </Link>
+    </Text>{' '}
+    and acknowledge you've read our
+    <Text size="sm" inline>
+      <Link href="https://www.pinterest.com" inline>
+        {' '}
+        Privacy Policy{' '}
+      </Link>
+    </Text>
+  </Text>
+</Flex>;
+`}
+          />
+          <MainSection.Card
+            type="don't"
+            description="Apply multiple text styles on links paired with text, as it can create inconsistency it can make it hard to scan. See the [Link and color variant](/link#Link-and-Text) for guidelines."
+            defaultCode={`
+<Flex direction="column" width="70%" gap={3}>
+  <Flex.Item>
+    <Box display="visuallyHidden" width="100%">
+      <Label htmlFor="example-email-2">Email</Label>
+    </Box>
+    <TextField placeholder="Email" id="example-email-2" onChange={() => {}} type="email" value="" />
+  </Flex.Item>
+  <Flex.Item>
+    <Box display="visuallyHidden">
+      <Label htmlFor="example-password-2">Password</Label>
+    </Box>
+    <TextField
+      placeholder="Create password"
+      id="example-password-2"
+      onChange={() => {}}
+      type="password"
+      value=""
+    />
+  </Flex.Item>
+  <Flex.Item>
+    <Box display="visuallyHidden">
+      <Label htmlFor="example-age-2">Age</Label>
+    </Box>
+    <NumberField placeholder="Age" id="example-age-2" onChange={() => {}} value="" />
+  </Flex.Item>
+  <Button fullWidth text="Create account" size="md" color="red" />
+  <Button fullWidth text="Log into existing account" size="md" type="submit" />
+  <Text size="sm" align="center">
+    By continuing, you agree to Pinterest's
+    <Text size="sm" inline weight="bold">
+      <Link href="https://www.pinterest.com" inline>
+        {' '}
+        Business Terms of Service{' '}
+      </Link>
+    </Text>{' '}
+    and acknowledge you've read our
+    <Text size="sm" inline weight="bold">
+      <Link href="https://www.pinterest.com" inline>
+        {' '}
+        Privacy Policy{' '}
+      </Link>
+    </Text>
+  </Text>
+</Flex>;
+`}
+          />
+        </MainSection.Subsection>
+        <MainSection.Subsection columns={2}>
+          <MainSection.Card
+            type="do"
+            description={`\`isExternal\` PROP COMING SOON.
```

Comment:
😅 Let's remove this do/don't pair for now?

Reply:
DONE